test/test_one_dimensional_shallow_water_model.jl

```diff
@@ -0,0 +1,49 @@
+# # One dimensional shallow water example
+
+using Oceananigans, Oceananigans.Advection
+
+using Oceananigans.Models # should import the shallow water model stuff
+#include("../src/Models/ShallowWaterModels/ShallowWaterModels.jl")
+using .ShallowWaterModels: ShallowWaterModel
+
+grid = RegularCartesianGrid(size=(64, 1, 1), extent=(2π, 2π, 2π))
+
+model = ShallowWaterModel(        grid = grid,
+                          architecture = CPU(),
+                             advection = nothing, 
+                              coriolis = nothing, 
+                             solution = nothing,
+                              tracers = nothing#,
+#                          timestepper = RungeKutta3 
+                                  )
+
+width = 0.3
+ h(x, y, z)  = 1.0 + 0.1 * exp(-(x - π)^2 / (2width^2)); 
+uh(x, y, z) = 0.0
+vh(x, y, z) = 0.0 
+
+include("../src/Models/ShallowWaterModels/set_shallow_water_model.jl")
```

Comment:
We shouldn't include a file from the source here

Reply:
I moved this and the subsequent include files above with the other include line